front/lib/plans/subscription.ts

```diff
@@ -234,6 +238,14 @@ export const internalSubscribeWorkspaceToFreePlan = async ({
   });
 };
 
+export const pokeUpgradeWorkspaceToEnterprise = async (
+  auth: Authenticator,
+  newPlanData: NewEnterpriseSubscriptionType
+) => {
+  console.log(auth, newPlanData);
```

Comment:
😱 ✂️

Reply:
This will be killed in the next PR, this is just to call the parameters so that typescript is happy since the method is not implemented yet :)